### DISCUSSION TITLE
Error at DEM visualisation step

### DISCUSSION THREAD

**Discussion by PriyomRoy** [alexisInSAR/EZ-InSAR#45] [Help]
DEM VISUALISATION STOPS

Error using xml2struct
The file /home/gsg/Joshimath/dem/dem.wgs84.vrt.xml could not be found

Error in isceprocessing (line 233)
        xmldem = xml2struct([pathdem,'.vrt']);

Error in GUIMIESAR>@(src,evt,arg1,arg2)isceprocessing(src,evt,'checkingDEM',figmiesar.UserData) (line 280)
bt_visualiseDEM_isceprocessing.ButtonPushedFcn = @(src,evt,arg1,arg2) isceprocessing(src,evt,'checkingDEM',figmiesar.UserData);
 
Error using matlab.ui.control.internal.controller.ComponentController/executeUserCallback
Error while evaluating Button PrivateButtonPushedFcn.

**Comment by alexisInSAR** [comment]
Please see the discussion [here](https://github.com/alexisInSAR/EZ-InSAR/discussions/24)

Kind regards, 

Alexis
